### ISSUE TITLE
Stuck on loading. 

### ISSUE THREAD

**Issue by alexsquared** [FlatPepsi17/MMM-WeatherGraph#10]
Module used to work flawlessly.  Had to do a fresh install after MM hung up. Installed module and copied app key and app now just stays stuck on loading.  I went into the Jason file and changed the app url from 2.5 to 3.0 based on forum recommendations and it did not work.   Then did module updates when prompted and still doesn't work.

**Comment by FlatPepsi17** [comment]
The weather API company change their policies. You need to go into where you got your API key, and sign up for their one call plan. You will need to give a credit card, but you can set the API limit so you never get billed.On Nov 1, 2024, at 8:26 PM, alexsquared ***@***.***> wrote:﻿
Module used to work flawlessly.  Had to do a fresh install after MM hung up. Installed module and copied app key and app now just stays stuck on loading.  I went into the Jason file and changed the app url from 2.5 to 3.0 based on forum recommendations and it did not work.   Then did module updates when prompted and still doesn't work.

**Comment by alexsquared** [comment]
Thank you for the quick reply.    I already had an API key through them and typed into the module.    I did just go to open weather and sign up for their OneCall service.  I also limited it to 1000 calls per day.   Is there anything I need to do in the config file in order for it to start working again?    I did modify the config file so that the interval for the calls is every 30 minutes (1800000 or 30*60*1000).    Do I need to do that in the json file that is inside the modules folder as well or just in my main config?   this should make it so it only calls about 720/day.

**Comment by FlatPepsi17** [comment]
Yes, that should be it.  When you set the API limit, you're making sure you
never get a bill. Editing the main config to 30 minutes should be just
fine- but if you happen to run over your limit, you just won't get fresh
data the rest of the day. (Like if you had multiple mirrors running
with the same API key)

 I did notice that when I signed up for my plan, it took half a day for the
new plan to become active.

**Comment by alexsquared** [comment]
Thank you for the help.   all is working as it should now.
